From ff72e7f248cea87eb0c8158ef847bc400a86fe12 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Sun, 1 Jun 2025 09:45:29 -0400
Subject: [PATCH 074/158] winewayland: add opcode 3 of
 zwlr_data_control_device_v1_listener.

---
 dlls/winewayland.drv/wayland_data_device.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/dlls/winewayland.drv/wayland_data_device.c b/dlls/winewayland.drv/wayland_data_device.c
index e36c0cc9229..429baa2f2f8 100644
--- a/dlls/winewayland.drv/wayland_data_device.c
+++ b/dlls/winewayland.drv/wayland_data_device.c
@@ -585,11 +585,19 @@ static void data_control_device_finished(
 {
 }
 
+static void data_control_device_primary_selection(
+    void *data, struct zwlr_data_control_device_v1 *zwlr_data_control_device_v1,
+    struct zwlr_data_control_offer_v1 *id
+)
+{
+}
+
 static const struct zwlr_data_control_device_v1_listener data_control_device_listener =
 {
     data_control_device_data_offer,
     data_control_device_selection,
     data_control_device_finished,
+    data_control_device_primary_selection
 };
 
 /**********************************************************************
-- 
2.49.0

